helix-term/src/ui/statusline.rs

```diff
@@ -372,15 +376,15 @@ where
     let position = get_position(context);
     write(
         context,
-        format!(" {}:{} ", position.row + 1, position.col + 1).into(),
+        format!(" {:>4}:{:<3} ", position.row + 1, position.col + 1).into(),
     );
 }
 
 fn render_total_line_numbers<'a, F>(context: &mut RenderContext<'a>, write: F)
 where
     F: Fn(&mut RenderContext<'a>, Span<'a>) + Copy,
 {
-    let total_line_numbers = context.doc.text().len_lines();
+    let total_line_numbers = context.doc.text().len_lines().saturating_sub(1);
```

Comment:
This would change the meaning of this element from being the count of lines to being the last line number